recipes/influxdb-cpp/all/test_package/conanfile.py

```diff
@@ -1,6 +1,8 @@
 import os
```

Comment:
The test package is mixing V1 with V2, also, self.setting should not be passed to cross_building anymore.